pkg/operator/ceph/pool/radosnamespace/controller.go

```diff
@@ -379,6 +388,11 @@ func (r *ReconcileCephBlockPoolRadosNamespace) deleteRadosNamespace(cephBlockPoo
 	namespacedName := fmt.Sprintf("%s/%s", cephBlockPoolRadosNamespace.Namespace, cephBlockPoolRadosNamespace.Name)
 	logger.Infof("deleting ceph blockpool rados namespace object %q", namespacedName)
 
+	if getRadosNamespaceName(cephBlockPoolRadosNamespace) == "" {
+		logger.Info("can't delete empty radosnamepace as it present by default.")
```

Comment:
Log messages should not end in periods.

Log messages should include enough info about the current underlying resource and/or the kube object namespace and name to make sure it is clear when debugging which resource is being referenced. Please add enough info to the info output to ensure the log can be correlated to the right inputs and outputs without guessing. 

I would suggest including both the CephBlockPoolRadosNamespace <namespace>/<name> and underlying Ceph namespace to ensure debugging can be done quickly without too much lookup.